src/main/java/com/pmm/budgetcalculator/expensing/domain/expense/Expense.java

```diff
@@ -0,0 +1,28 @@
+package com.pmm.budgetcalculator.expensing.domain.expense;
+
+import com.pmm.budgetcalculator.foundation.BaseEntity;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import javax.persistence.Entity;
+import java.time.LocalDateTime;
+
+
+@Entity
+@Data
+@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)
+@AllArgsConstructor
+
+public class Expense extends BaseEntity {
+    long expenseId;
```

Comment:
Here is long and in Controller requestParam is Long - use 1 approach. Which one? https://stackoverflow.com/questions/9146967/should-i-use-primitives-or-wrappers-in-jpa2-0
Also, is id in base entity and expenseId 2 different things?

Reply:
I choose "wrapped" Long.
It's the same I think, so should I delete expenseId in Class Expense? It means only Id in BaseEntity is enough?

Reply:
Yes, ID in BaseEntity is enough.